lib/ansible/modules/system/aix_chsec.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: aix_chsec
+
+short_description: modify aix stanza files
+
+version_added: "0.1"
+
+description:
+    - "adds stanzas to aix config files using the chsec command. see "man chsec" for additional infos"
```

Comment:
```suggestion
    - Modify stanzas to AIX config files using the chsec command.
notes:
    - See `man chsec` for additional information.
```
This fixes an important documentation syntax error, causing a lot of errors as a result.